notes/10_svm/2_margin.tex

```diff
@@ -265,7 +265,7 @@ \subsection{Margins and the VC-dimension}
          }
      \end{subfigure}
      \mode<article>{
-     \caption{A linear neuron can shatter any label configuration of 3 points regardless of how they are arranged in 2D space (excluding colinear points).}
+     \caption{A linear neuron can shatter any label configuration of 3 points regardless of how they are arranged in 2D space (excluding collinear points).}
```

Comment:
I thought you were going for AE spelling

Reply:
My bad, didn't recognize this as a valid spelling. I only recalled that you had used "collinear" in previous documents.

Reply:
You're right, it's colinear here and collinear [here](https://github.com/kashefy/mi1notes/blob/master/notes/09_slt/1_linsep.tex#L35). So the latter would have to change to a single 'l' to be consistent with AE.

Reply:
I will take care of that next time my brain hurts from studying.

Reply:
No worries, will take care of the collinear/colinear bit.